core/adapters/http.go

```diff
@@ -64,6 +85,11 @@ type HTTPPost struct {
 	QueryParams  QueryParameters `json:"queryParams"`
 	Body         *string         `json:"body,omitempty"`
 	ExtendedPath ExtendedPath    `json:"extPath"`
+	timeout      *string         `json:"timeout,omitempty"`
```

Comment:
Probably should document how this impacts the post body.

Reply:
How does it impact the POST body>?

Reply:
If my understanding is correct, the timeout is enforced no matter what, so if you ship a large body and that takes longer to send than the timeout, this will fail.

Reply:
Aaaaah. I will add this to the documentation 👍 

I don't think it's too much of an issue in practice, since the default timeout of 5s is fairly lenient and its unlikely that extremely large bodys will be submitted from on-chain.

Reply:
Yup, just a fyi.

Reply:
Maybe this type may help?  I agree discoverability of these things can be improved.

https://github.com/smartcontractkit/chainlink/blob/develop/core/store/models/common.go#L403

Reply:
Where do you imagine we would document this @j16r ?